static/css/foundation.css

```diff
@@ -47,7 +47,12 @@ footer,
 header,
 nav,
 section {
-  display: block; }
+  display: block;
```

Comment:
Modifying the foundation css is a no-go. Next upgrade these changes will be gone and it'll likely be broken again.

Reply:
Understood
That's why I changed only app.css.

Reply:
@sanchaymittal Your PR still has changes in `foundation.css`. Since you've made the necessary changes in `app.css`, please revert foundation to the original state.

Also, could you please attach pre/post fix screenshots (of mobile device too!)?

Reply:
I was aware of that fact that's why I arranged a new pull request regarding the  same problem.

Reply:
https://github.com/CCExtractor/sample-platform/issues/165#issuecomment-366498316